lib/timex/types.ex

```diff
@@ -29,6 +29,7 @@ defmodule Timex.Types do
   @type weekday_name :: :monday | :tuesday | :wednesday | :thursday | :friday | :saturday | :sunday
   @type shift_units :: :milliseconds | :seconds | :minutes | :hours | :days | :weeks | :years
   @type time_units :: :microseconds | :milliseconds | :seconds | :minutes | :hours | :days | :weeks | :years
+  @type second_time_units :: :second | :millisecond | :microsecond | :nanosecond | pos_integer() | :seconds | :milliseconds | :microseconds | :nanoseconds
```

Comment:
I'm probably missing something, but where does the pos_integer function come from?

Reply:
Good catch! I got it from [time_unit](https://hexdocs.pm/elixir/System.html#t:time_unit/0) that is used on `System.convert_time_unit/3`.

But seems that [DateTime.from_unix!](https://github.com/elixir-lang/elixir/blob/v1.8.1/lib/elixir/lib/calendar/datetime.ex#L165) does not work with positive integers despite its typespec.

Fixed. Thank you!